src/platforms/stub/application/core_application_stub.cpp

```
@@ -0,0 +1,33 @@
+// DEALINGS IN THE SOFTWARE.
+//
+/*!
+  \file   core_application_stub.cpp
+  \brief  Stub implementation of \ref toy::application::CoreApplication::sleep() for platforms without OS sleep support.
+*/
+
+#include "application.hpp"
+
+namespace toy::application {
+
+void CoreApplication::sleep([[maybe_unused]] size_t milliseconds) const noexcept {
```

Cppcheck (2.20.0)

[style] The function 'sleep' is never used.

(unusedFunction)